transforms/helpers/options.js

```diff
@@ -0,0 +1,13 @@
+const { getOptions } = require('codemod-cli');
+
+module.exports = {
+  getCoercedOptions(keys) {
```

Comment:
Thinking about it I'm going to see if I can push a better fix upstream to codemod-cli